### ISSUE TITLE
Handling of non-ASCII characters (Django issue)

### ISSUE THREAD

**Issue by fmakowski** [RyanNoelk/OpenEats#305]
Hey,

this issue has a low priority, but decided to make a ticket just to ensure it's acknowledged. It seems that Django (Python2 version) is not handling non-ASCII characters well, and by that I mean UTF-8 charset (like Polish śćń etc., but also Spanish ñ). The issue and its solutions are described well in [that SO thread](https://stackoverflow.com/questions/4635188/django-non-ascii-character). 

The result is that API responds with 500 internal error code, both via front-end panel, as the built-in Django admin panel.

I volunteer to revisit that issue next month and create a pull request that solves it. 

Best regards!
Filip

**Comment by RyanNoelk** [comment]
I'm not having this issue. I was able to add a few recipes that had umlauts in them just fine. Was there a specific part of the recipe creation that failed for you.

As a side note, all python files should already have the utf-8 encoding header.

**Comment by fmakowski** [comment]
Yeah, ü and other umlauts work just fine. I am not yet sure what might be the cause, still, I will try to work on that.

**Comment by RyanNoelk** [comment]
Your totally right. I always use umlauts as my default utf-8 test. Let me know what you find out ^^

**Comment by RyanNoelk** [comment]
I just created a new instance of the code and I'm not seeing this issue (even with the chars you mentioned above). This could be a database issue.
